ej2-react/check-box/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with React Check box component | Syncfusion
-description:  Checkout and learn about Getting started with React Check box component of Syncfusion Essential JS 2 and more details.
+title: Getting started with React Checkbox Component | Syncfusion
+description:  Checkout and learn about Getting started with Syncfusion Essential React Checkbox component, its elements and more.
```

Comment:
Revert these description changes and remove extra spaces.

Reply:
Reverted